server/src/main/java/com/objectcomputing/checkins/services/pulseresponse/PulseResponseController.java

```diff
@@ -56,17 +58,17 @@ public PulseResponseController(PulseResponseService pulseResponseServices,
         this.ioExecutorService = ioExecutorService;
     }
 
-    @Error(exception = PulseResponseBadArgException.class)
-    public HttpResponse<?> handleBadArgs(HttpRequest<?> request, PulseResponseBadArgException e) {
+    @Error(exception = BadArgException.class)
+    public HttpResponse<?> handleBadArgs(HttpRequest<?> request, BadArgException e) {
         JsonError error = new JsonError(e.getMessage())
                 .link(Link.SELF, Link.of(request.getUri()));
 
         return HttpResponse.<JsonError>badRequest()
                 .body(error);
     }
 
-    @Error(exception = PulseResponseNotFoundException.class)
-    public HttpResponse<?> handleNotFound(HttpRequest<?> request, PulseResponseNotFoundException e) {
+    @Error(exception = NotFoundException.class)
+    public HttpResponse<?> handleNotFound(HttpRequest<?> request, NotFoundException e) {
```

Comment:
you can remove this. It will come from the exceptions folder

Reply:
Specifically `com.objectcomputing.checkins.services.exceptions.handlers.NotFoundHandler`. Not sure when the change to global error handlers was made but it's a good change!

Reply:
updated